src/views/v2/components/breadcrumbs/linked-breadcrumb.php

```diff
@@ -11,21 +12,40 @@
  *
  * @link http://evnt.is/1aiy
  *
- * @version 6.2.0
+ * @version TBD
  * @since 6.2.0 Added classes and title attribute to the anchor tags.
+ * @since TBD Added W3C ARIA breadcrumb attributes for accessibility.
  *
- * @var array $breadcrumb Data for breadcrumb.
+ * @var array $breadcrumb {
+ *                        *     Breadcrumb data.
+ *                        *
+ *                        *     @type string $label Breadcrumb text.
+ *                        *     @type string $link URL for the breadcrumb link.
+ *                        *     @type string $title Optional title attribute for the link.
+ *                        *     @type bool $is_last True if this is the last breadcrumb.
+ *                        * }
  */
-$title = Arr::get( 'title', '' );
+
+if ( empty( $breadcrumb['label'] ) || empty( $breadcrumb['link'] ) ) {
+	return;
+}
+
+$crumb_title = Arr::get( $breadcrumb, 'title', '' );
+$crumb_link  = $breadcrumb['link'];
+$crumb_label = $breadcrumb['label'];
```

Comment:
I don't know that these variable add anything other than a bit of neatness to the template code. Don't think it needs to change, but they feel like "useless variables" to me.